Source/Scene/PointCloud.js

```diff
@@ -819,8 +823,13 @@ define([
                 if (!defined(clippingPlanes)) {
                     return Matrix4.IDENTITY;
                 }
-                var modelViewMatrix = Matrix4.multiply(context.uniformState.view3D, pointCloud._modelMatrix, scratchClippingPlaneMatrix);
-                return Matrix4.multiply(modelViewMatrix, clippingPlanes.modelMatrix, scratchClippingPlaneMatrix);
+
+                if (!defined(pointCloud._clippingPlaneOffsetMatrix)) {
+                    Matrix4.multiply(context.uniformState.view3D, pointCloud._modelMatrix, scratchClippingPlaneMatrix);
+                } else {
+                    Matrix4.multiply(context.uniformState.view3D, pointCloud._clippingPlaneOffsetMatrix, scratchClippingPlaneMatrix);
+                }
+                return scratchClippingPlaneMatrix;
```

Comment:
Is `clippingPlanes.modelMatrix` getting accounted for?

Reply:
This may explain the failing test: `Scene/TimeDynamicPointCloud sets clipping planes`

Reply:
Yes, that's my bad! That shouldn't have been removed. Thanks for catching this.